sycl/test-e2e/ESIMD/vadd_raw_send.cpp

```diff
@@ -84,7 +84,7 @@ ESIMD_INLINE void block_write2(AccessorTy acc, unsigned int offset,
   auto src0_ref2 = src0.template select<8, 1>(8);
 
   src0_ref1.template select<1, 1>(2) = offset >> 4;
-  src0_ref2 = data;
+  src0_ref2 = data.template bit_cast_view<unsigned int>();
```

Comment:
The variable src0_ref2 is unused, It has 2 defines and 0 uses.

Reply:
Ok, I understand. Thank you.
So, the second assignment works as a write to the memory pointed by LHS. IMO this case potentially shows we have some design/api issue because as the code is written here looks pretty misleading/confusing.